This patch fixes a build issue on solaris.
It is intended to be submitted upstream by 2016-Jul-01.
--- openscap-1.2.9/src/OVAL/oval_session.c.~1~	2016-05-02 11:04:04.741724115 -0700
+++ openscap-1.2.9/src/OVAL/oval_session.c	2016-05-02 11:04:20.852019147 -0700
@@ -27,7 +27,7 @@
 #include <libgen.h>
 #include <unistd.h>
 #include <string.h>
-#include <linux/limits.h>
+#include <limits.h>
 
 #include "common/alloc.h"
 #include "common/debug_priv.h"
